apps/web/src/index.html

```diff
@@ -16,7 +16,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" type="image/svg+xml" href="/img/logo.svg" />
```

Comment:
this is not fully supported by [caniuse](https://caniuse.com/link-icon-svg) we should use [realfavicongenerator](https://realfavicongenerator.net/) to generate our favicons for multiple devices.